code/modules/mob/living/carbon/xenomorph/powers.dm

```diff
@@ -925,6 +925,10 @@ GLOBAL_LIST_INIT(acid_spray_hit, typecacheof(list(/obj/structure/barricade, /obj
 		face_atom(C)
 		if(stagger)
 			return FALSE
+		body_tox = C.reagents.get_reagent(toxin)
+		if(CHECK_BITFIELD(C.status_flags, XENO_HOST) && body_tox && body_tox.volume > body_tox.overdose_threshold)
```

Comment:
should this maybe be critical overdose threshold

Reply:
That might increase the number of accidental OD deaths and change the dynamic of the chemical.
I understand the issue with basically ignoring the critical obverdose effects, because they would kind of not appear in game anymore, but I believe it's pretty clear that they are not needed and just cause salt with capture kills instead.